modules/nw-ipfailover-cluster-ha-ingress.adoc

```diff
@@ -3,10 +3,10 @@
 // * networking/configuring-ipfailover.adoc
 
 [id="nw-ipfailover-cluster-ha-ingress_{context}"]
-= High availability For ingressIP
+= High availability For ExternalIP
 
-In non-cloud clusters, IP failover and `ingressIP` to a service can be combined. The result is high availability services for users that create services using `ingressIP`.
+In non-cloud clusters, IP failover and `ExternalIP` to a service can be combined. The result is high availability services for users that create services using `ExternalIP`.
 
-The approach is to specify an `ingressIPNetworkCIDR` range and then use the same range in creating the ipfailover configuration.
+The approach is to specify an `spec.ExternalIP.autoAssignCIDRs` range and then use the same range in creating the IP failover configuration.
 
-Because IP failover can support up to a maximum of 255 VIPs for the entire cluster, the `ingressIPNetworkCIDR` needs to be `/24` or smaller.
+Because IP failover can support up to a maximum of 255 VIPs for the entire cluster, the `spec.ExternalIP.autoAssignCIDRs` must be `/24` or smaller.
```

Comment:
I think it'd be good to clarify that `spec.ExternalIP.autoAssignCIDRs` is a field of the cluster network config. If I was reading this without additional context, I'd think this was referring to the ip failover deployment's `spec`.

I'm not sure if there's a style guideline for referring to other cluster resources, but the relevant resource is `network.config/cluster`, and some info on it is [here](https://docs.openshift.com/container-platform/4.15/networking/cluster-network-operator.html#nw-cno-view_cluster-network-operator), if that's helpful.